diff --git a/src/shared/Database/AutoUpdater.cpp b/src/shared/Database/AutoUpdater.cpp
index 8a9debc..850e584 100644
--- a/src/shared/Database/AutoUpdater.cpp
+++ b/src/shared/Database/AutoUpdater.cpp
@@ -385,7 +385,7 @@ namespace DBUpdater
         auto authUpdateFolder = sConfig.GetStringDefault("Database.AutoUpdate.AuthUpdateName", "Logon");
         auto charUpdateFolder = sConfig.GetStringDefault("Database.AutoUpdate.CharUpdateName", "Char");
         auto worldUpdateFolder = sConfig.GetStringDefault("Database.AutoUpdate.WorldUpdateName", "World");
-        bool sortByName = sConfig.GetBoolDefault("Database.AutoUpdate.SortByName", false);
+        bool sortByName = true;
         path folderPath{ pathString };
 
 
