diff --git a/src/shared/Database/AutoUpdater.cpp b/src/shared/Database/AutoUpdater.cpp
index 19fc222..c14a1a6 100644
--- a/src/shared/Database/AutoUpdater.cpp
+++ b/src/shared/Database/AutoUpdater.cpp
@@ -137,19 +137,9 @@ namespace DBUpdater
             }
         }
 
-        //sort by modified-at ascending for oldest -> newest updates
         std::sort(updates.begin(), updates.end(), [](const FileMigration& e1, const FileMigration& e2)
         {
-            if (e1.ModifiedAt < e2.ModifiedAt)
-                return true;
-
-            if (e1.ModifiedAt > e2.ModifiedAt)
-                return false;
-
-
-            //if we're here then both modifiedAt are the same, can happen with Git merges etc. We sort alphabetically by lowest -> oldest first.
             return e1.Name < e2.Name;
-
         });
 
         for (const auto& update : updates)
